src/System.Text.Encodings.Web/src/project.json

```diff
@@ -1,16 +1,16 @@
 {
   "dependencies": {
-    "System.Collections": "4.0.10",
+    "System.Collections": "4.0.0",
     "System.Diagnostics.Contracts": "4.0.0",
-    "System.Diagnostics.Debug": "4.0.10",
+    "System.Diagnostics.Debug": "4.0.0",
     "System.Diagnostics.Tools": "4.0.0",
-    "System.Globalization": "4.0.10",
-    "System.IO": "4.0.10",
-    "System.Reflection": "4.0.10",
+    "System.Globalization": "4.0.0",
+    "System.IO": "4.0.0",
+    "System.Reflection": "4.0.0",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Runtime.Extensions": "4.0.10",
+    "System.Runtime": "4.0.10",
```

Comment:
You can go back as far as 4.0.0 and have this work on net 4.5 as well.  Make sure to do so for all dependencies.